presets/4.3/vtx/diatone_mamba_ultra_1000.txt

```diff
@@ -0,0 +1,52 @@
+#$ TITLE: Diatone Mamba Ultra 1000 VTX Tables
+#$ FIRMWARE_VERSION: 4.2
+#$ FIRMWARE_VERSION: 4.3
+#$ CATEGORY: VTX
+#$ STATUS: COMMUNITY
+#$ KEYWORDS:  vtx, vtx table, diatone, mamaba, ultra 1000
+#$ AUTHOR: NemesisXB
+#$ DESCRIPTION: VTX tables for the Diatone Mamba Ultra 1000.
+#$ DESCRIPTION: Select the the HAM option if you have a HAM license or the NO HAM if you do not.
+#$ DESCRIPTION: EU users select the NO HAM option.
+#$ DISCLAIMER:  Only select one option. All previous VTX Table settings will be reset.
+#$ INCLUDE_DISCLAIMER: misc/disclaimer/en/vtxtable.txt
+#$ DISCUSSION:  https://github.com/betaflight/firmware-presets/pull/107
+
+# reset basic Vtx settings
+set vtx_channel = 0
+set vtx_power = 0
+set vtx_low_power_disarm = OFF
+set vtx_freq = 0
+set vtx_pit_mode_freq = 0
+set vtx_halfduplex = ON
+set vtx_spi_bus = 0
+
+# reset VTx table
+vtxtable bands 0
+vtxtable channels 0
+vtxtable powerlevels 0
+vtxtable powervalues
+vtxtable powerlabels
```

Comment:
as above

Reply:
This not only clears the vtx table but also the vtx stuff:
vtx 0 0 0 0 0 900 900
vtx 1 0 0 0 0 900 900
vtx 2 0 0 0 0 900 900
vtx 3 0 0 0 0 900 900
Which I don't think it should.

Reply:
why not? the idea is to reset every thing... that's why its a default file

Reply:
@limonspb  what are your thoughts

Reply:
> why not? the idea is to reset every thing... that's why its a default file

I will add it if this holds up the merge. But my thought was that the user probably already has this configured in a way I would not be able to configure on his behalf. If a user for example swaps a vtx out, he would still want the VTX settings configured the same, what could happen is that the new vtx might not have the same number of power levels and the user might have to correct that, but loading the table from the json file would have had the same issue.

In the end using this preset is like using the json file. You just don't have to go and look for it and download it.

Reply:
> why not? the idea is to reset every thing... that's why its a default file

Also, as per the readme:
"Ignore some values, or not change values that don't matter to your Preset, at whatever value the user the user has in their configuration, by not mentioning those parameters in the Preset."

Reply:
Hi, @NemesisXB. Thanks for your pull request.
I understand why you do not want to reset 
vtx 0 ... and vtx 1... settings. It makes sense.
But at the same time, why do you want to reset these settings?
```
set vtx_low_power_disarm = OFF
set vtx_pit_mode_freq = 0
set vtx_halfduplex = ON
```

Reply:
> Hi, @NemesisXB. Thanks for your pull request. I understand why you do not want to reset vtx 0 ... and vtx 1... settings. It makes sense. But at the same time, why do you want to reset these settings then?
> 
> ```
> set vtx_low_power_disarm = OFF
> set vtx_pit_mode_freq = 0
> set vtx_halfduplex = ON
> set vtx_spi_bus = 0
> ```
> 
> you do not have to change anything yet, let's discuss :)

I'm 100% for not changing these as well. I actually copied everything from the default file except the vtx 0 ... and vtx 1 ... stuff, as I do not really know the other options.

I am not 100% sure about the vtx_halfduplex. This VTX uses IRC tramp which I assume is half duplex (as it has a single data wire) so setting the vtx_halfduplex to on still seems OK. And vtx_spi_bus, I don't know what this is for, I assume vtx controlled via spi? But the first 2 I also think should not be changed by a vtx table preset.

In the end, you probably know better than me what a user might want and not want. And I will update the PR to your requirement. You are after all the guys who put in all the work, and if I'm not mistaken, presets is your baby.

Reply:
@NemesisXB  ive just created a vtxtable only default. once its merged lets use that..

Reply:
@NemesisXB
Please have a look at this pull request:
https://github.com/betaflight/firmware-presets/pull/109
see if you think that these defaults would be cool for VTX tables.

Reply:
> @NemesisXB Please have a look at this pull request: #109 see if you think that these defaults would be cool for VTX tables. Then we merge it and you can use new defaults that resets ONLY vtx tables.

@limonspb, Looks good to me. Will update as soon as it is merged.

Reply:
@NemesisXB tiz merged